File: ClientApp/src/app/models/notify-message.ts
/** represent chat message class */
export class NotifyMessage {

    user: string;
    message: string;
    room: string;

    constructor(user: string = '', message: string = '', room: string = '') {
      this.user = user;
      this.message = message;
      this.room = room;
    }
  }

File: Models/NotifyMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace xSignalRng6.Models
{
    public class NotifyMessage
    {
        public string user { get; set; }

        public string message { get; set; }

        public string room { get; set; }
    }
}
